src/components/AppointmentScheduler.tsx

```diff
@@ -1,47 +1,92 @@
 import React, { useState, useEffect } from "react";
```

Comment:
Please consider adding documentation to this file. Maybe a quick docstring-esque description to the subroutines defined and called in here?